src/components/tree-views-container.js

```diff
@@ -578,7 +578,7 @@ const TreeViewsContainer = () => {
                     }
                 >
                     {/* Directories Menu */}
-                    {!showMenuFromEmptyZone && (
```

Comment:
strange fix. anchorEl is already used by the StyledMenu open prop, it should be ok.
The `showMenuFromEmptyZone` is suspicious (I added it :P)